From 83124a7df40d08b7731842a866adb7249cd44d13 Mon Sep 17 00:00:00 2001
From: Olivier Fourdan <ofourdan@redhat.com>
Date: Fri, 25 Oct 2024 16:46:40 +0200
Subject: [PATCH 04/10] window: Use the fullscreen API

Use the meta_window_is_fullscreen() API instead of accessing the
fullscreen field of the MetaWindow structure directly.

This is both a (small) cleanup and preparation work for the next commit.

No function change (intended).

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/4076>
(cherry picked from commit a7ef0b137e194f970975e2653e89630df260348f)
---
 src/core/window.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/window.c b/src/core/window.c
index 1782c884fa..d803fbdc1d 100644
--- a/src/core/window.c
+++ b/src/core/window.c
@@ -368,7 +368,7 @@ meta_window_get_property(GObject         *object,
       g_value_set_boolean (value, win->decorated);
       break;
     case PROP_FULLSCREEN:
-      g_value_set_boolean (value, win->fullscreen);
+      g_value_set_boolean (value, meta_window_is_fullscreen (win));
       break;
     case PROP_MAXIMIZED_HORIZONTALLY:
       g_value_set_boolean (value, win->maximized_horizontally);
-- 
2.49.0

